main.go

```diff
@@ -53,15 +150,15 @@ func main() {
 			event := Stream{}
 			decErr := json.Unmarshal(line, &event)
 			if decErr != nil {
-				log.Println("## Error decoding json", err)
+				log.Println("## Error decoding json.", err)
 				resp.Body.Close()
 				break
 			}
-
-			fmt.Printf("%v | Project: %v | Name: %v | Kind: %v | Reason: %v | Message: %v\n",
-				event.Event.LastTimestamp,
-				event.Event.Namespace, event.Event.Name,
+			// using "|" to seperate each field for easy parsing by log analyzers
```

Comment:
This pattern can not be changed, as we already specified this pattern on our logging infrastructure.

Reply:
Completely understand, I have updated to reflect the original formatting.